en/gps_compass/README.md

```diff
@@ -132,9 +132,9 @@ Parameter | Setting
 Compass calibration is covered in: [Compass Configuration](../config/compass.md).
 The process is straightforward and will calibrate all connected magnetometers.
 
-Additional configuration can be [performed](../advanced_config/parameters.md) using the [CAL\_MAGx\_](../advanced_config/parameter_reference.md#CAL_MAG0_EN) parameters (where `x=0-3`).
-Generally you will not need to *modify* these as compasses are autodetected, prioritised and are all calibrated at the same time (a possible exception is [CAL\_MAGx\_EN](../advanced_config/parameter_reference.md#CAL_MAG0_EN) which might be used to disable an internal compass).
-You may however wish to read them, as they will let you know which magnetometers are internal or external ([CAL\_MAGx\_EN](../advanced_config/parameter_reference.md#CAL_MAG0_EN)) and which is being uses as the main heading source ([CAL_MAG_PRIME](../advanced_config/parameter_reference.md#CAL_MAG_PRIME)).
+Additional configuration can be [performed](../advanced_config/parameters.md) using the [CAL\_MAGx\_](../advanced_config/parameter_reference.md#CAL_MAG0_ID) parameters (where `x=0-3`).
+Generally you will not need to *modify* these as compasses are autodetected, prioritised and are all calibrated at the same time (a possible exception is [CAL\_MAGx\_PRIO](../advanced_config/parameter_reference.md#CAL_MAG0_PRIO) which might be used to disable an internal compass).
+You may however wish to read them, as they will let you know which magnetometers are internal or external ([CAL_MAGn_ROT==-1](../advanced_config/parameter_reference.md#CAL_MAG0_ROT ) indicates internal) and which is being uses as the main heading source ([CAL_MAG_PRIME](../advanced_config/parameter_reference.md#CAL_MAG_PRIME)).
```

Comment:
@bkueng I did minor update to this to replace CAL_MAG0_EN with CAL_MAG0_PRIO, and state that  you might use CAL_MAGn_ROT==1 to indicate that the compass is internal/external. 

However:
1. Not sure it is correct - is it true that prioritisation is done by default, along with selection of the prime compass? 
2. `CAL_MAG_PRIME` is gone. IS there no way to get the prime compass? If not, just delete the last sentence "and which is being uses as the main heading ..."?

Reply:
Yes external ones will have a higher prio. You can change that as you want to select a different one as primary.

Reply:
@bkueng Ah, so essentially everything gets done automatically but you can play with these yourself after that? When do they reset? i.e. say I add a new compass, does that cause everything to reshuffle? What if I fix one external mag to highest priority - will that ever be autoshuffled?

Reply:
Exactly. Existing ID's don't get changed, new ones will be assigned a new cal param index. What happens on 'overflow' is a question for @dagar.

Reply:
Thanks @bkueng . I have reworded a little to make this clear.

MY assumption here is that the auto-calibration is pretty smart - it prioritizes external over internal compasses, and it uses some sort of algorithm to determine which compasses are better than others. Further the internal compass will always be the lowest priority so would never actually get used? So why would you ever need to change the priority or disable an internal compass? Perhaps because you know they'd be better off having no compass than using the internal one?
Similarly, is there a chance they will get the priority better than the internal algorithm?

Perhaps you might want to disable an external compass during testing to work out whether it is actually needed? Doing it in software being easier than physically removing it?

The reason I ask is that if we can be explicit that would be better - e.g. have a heading "disable internal compass" and "changing the priority" makes sense if these are thinks we ever think people need to do - because that will help them find the instructions.

Anyway, merging this for now, but will revisit based on your comment.

Reply:
Almost as smart as you would expect: all external ones have the same prio (same for internal). Internal ones are used on failover. So yes, you might prefer to not use that at all.

> Perhaps you might want to disable an external compass during testing to work out whether it is actually needed?

Yes, another possible use case.
Probably in most cases you just want to disable the internal one(s).

Reply:
Thanks. FYI only I've captured this use case as the important one in https://github.com/PX4/PX4-user_guide/pull/1350